sspl_test/run_test.py

```diff
@@ -99,7 +99,8 @@ def tmain(argp, argv):
             try:
                 start_time = time.time()
                 if storage_type == 'virtual':
-                    if 'realstore' in test.__name__ or 'real_stor' in test.__name__:
+                    test_disable_module_substrings = ['realstore', 'real_stor', 'realstor', 'bmc']
```

Comment:
I thought we use some ipmi simulation, will not it be used here instead of disabling the bmc test?

Reply:
BMC code is a part of NodeHW sensor which gets disabled in case of virtual server_type. Hence, I think we need to disable this test as well.